packages/kbn-guided-onboarding/src/types.ts

```diff
@@ -6,14 +6,14 @@
  * Side Public License, v 1.
  */
 
-export type GuideId = 'observability' | 'security' | 'search' | 'testGuide';
+export type GuideId = 'kubernetes' | 'siem' | 'search' | 'testGuide';
```

Comment:
I'm not sure it makes sense to maintain these types now that the guide configs are registered within each plugin. WDYT?

Reply:
It was helpful to rely on type checking while I was refactoring the IDs but I agree we might want make them generic `string`s in the future. I think that could be done together with updating the examples plugin and functional tests early next release, wdyt? I can open an issue for that.